frontend/src/components/CreateRecipeForm/CreateRecipeForm.js

```diff
@@ -96,11 +98,11 @@ class CreateRecipeForm extends Component {
           <div className="col-6 offset-3">
             <div class="form-group">
               <label for="Directions"><h2>Directions</h2></label>
-              <input class="form-control" type="text" placeholder=""></input>
-              <input class="form-control" type="text" placeholder=""></input>
-              <input class="form-control" type="text" placeholder=""></input>
-              <input class="form-control" type="text" placeholder=""></input>
-              <input class="form-control" type="text" placeholder=""></input>
+              <input class="form-control" type="text" placeholder="" ></input>
+              <input class="form-control" type="text" placeholder="" ></input>
+              <input class="form-control" type="text" placeholder="" ></input>
+              <input class="form-control" type="text" placeholder="" ></input>
+              <input class="form-control" type="text" placeholder="" ></input>
```

Comment:
This kind of a question of taste, but I'm not sure we're using these before-closing spaces anywhere else. I personally find them odd, but maybe that's just my HTML1 biases showing through.